aptible/stack.go

```diff
@@ -0,0 +1,81 @@
+package aptible
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aptible/go-deploy/client/operations"
+)
+
+type Stack struct {
+	ID             int64
+	OrganizationID string
+	Name           string
+}
+
+func (s Stack) isShared() bool {
+	return s.OrganizationID == ""
+}
+
+func (c *Client) GetStacks() ([]Stack, error) {
+	page := int64(0)
```

Comment:
If we're exposing this method we'll need to require a page number (and return additional metadata about the results) or iterate through all pages and return every Stack.

Reply:
the only way for me to test this is to flood my account with stacks (?) and test pagination there, right?